tests/models/openconfig/data/openconfig_system/parse/ios/config/result.json

```diff
@@ -0,0 +1,98 @@
+{
+  "openconfig-system:system": {
+    "config": {
+      "hostname": "csr1000v-1"
+    },
+    "clock": {
+      "config": {
+        "timezone-name": "CST -6 0"
+      }
+    },
+    "dns": {
+      "servers": {
+        "server": [
+          {
+            "address": "8.8.8.8",
+            "config": {
+              "address": "8.8.8.8",
+              "port": 53
+            }
+          }
+        ]
+      }
+    },
+    "ntp": {
+      "config": {
+        "enabled": true,
+        "enable-ntp-auth": false
+      },
+      "servers": {
+        "server": [
+          {
+            "address": "2610:20:6F96:96::6",
+            "config": {
+              "address": "2610:20:6F96:96::6"
+            }
+          },
+          {
+            "address": "129.6.15.29",
+            "config": {
+              "address": "129.6.15.29"
+            }
+          }
+        ]
+      }
+    },
+    "ssh-server": {
+      "config": {
```

Comment:
Should we add another config to test the different SSH versions? And maybe NTP authenticate as well? Just the different scenarios that can be detected by the parsing?

Reply:
NTP auth is now done.  On the SSH version... anyone using SSH V1 with YANG... :man_shrugging: .  I'll gladly accept PR's if someone else wants to tackle that.  :smile: